server/controllers/userController.js

```diff
@@ -97,10 +97,41 @@ const register = async (req, res) => {
   }
 };
 
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    // check if user exists
+    const findUser = await findUserByEmail(email);
+
+    if (!findUser) {
+      return res.json({ error: 'Incorrect Email or Password' });
```

Comment:
Should add a response code in the 400 range to let the client know this was a bad request. Also might want to consider restructuring object. Most of our errors are returned as an error object with a message property. Consistency would help the front end handle errors with reusable lines of logic.